.github/ISSUE_TEMPLATE/plugin_request.md

```diff
@@ -7,24 +7,26 @@ labels: plugin-request
 <!--
 **NB:** Before opening a plugin support request against this repo, consider whether the plugin should/could be implemented in the [other OpenTelemetry client libraries](https://github.com/open-telemetry/). If so, please [open an issue on opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/issues/new) first.
 
-You are welcome to try out the [plugin api](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/plugin-guide.md) to build your own plugin. If you do try out the plugin api, please let us know if you have any questions/feedback.
+You are welcome to try to build your own plugin. If you do, please let us know if you have any questions/feedback.
 -->
 
-**Is your plugin request related to a problem? Please describe.**
+### Is your plugin request related to a problem? Please describe
 <!--
 A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
 -->
 
-**Is it applicable for Node or Browser or both**
 
-**Do you expect this plugin to be commonly used**
+### Is it applicable for Node or Browser or both
+
+
+### Do you expect this plugin to be commonly used
 Weekly Downloads:
 
-**What version of plugin are you interested in using**
+### What version of plugin are you interested in using
```

Comment:
```suggestion
### What version of plugin are you interested in using ?
```

Reply:
Also renaming plugin into instrumentation make sense

Reply:
Can do. Didn't because I thought that #427 might eventually do a global ctrl-f or something... I'll change that. Thanks!